packages/components/releaseNotes/labkey/components.md

```diff
@@ -1,6 +1,14 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version 0.70.6
```

Comment:
We usually leave this as `version TBD` up until right before the PR is merged since it is possible that someone else could get to merging in there changes first in which case this version number might not be correct.

Reply:
Fixed